include/votca/tools/unitconverter.h

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright 2009-2019 The VOTCA Development Team (http://www.votca.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#pragma once
+#ifndef VOTCA_TOOLS_UNITCONVERTER_H
+#define VOTCA_TOOLS_UNITCONVERTER_H
+
+#include <iostream>
+#include <map>
+namespace votca {
+namespace tools {
+
+enum DistanceUnit { meters, centimeters, nanometers, angstroms, bohr };
+
+enum MassUnit {
+  attograms,
+  picograms,
+  femtograms,
+  atomic_mass_units,
+  grams_per_mole,  // the same as atomic mass units
+  kilograms,
+  grams
+};
+
+enum TimeUnit { seconds, microseconds, nanoseconds, femtoseconds, picoseconds };
+
+enum EnergyUnit {
+  electron_volts,
+  kilocalories,
+  hartrees,
+  joules,
+  kilojoules,
+  kilojoules_per_mole,
+  joules_per_mole,
+  kilocalories_per_mole
+};
+
+enum ChargeUnit { e, coulombs };
+
+enum VelocityUnit {
+  angstroms_per_femtosecond,
+  angstroms_per_picosecond,
+  nanometers_per_picosecond
+};
+
+enum ForceUnit {
+  kilocalories_per_mole_ansgtrom,
+  newtons,
+  kilojoules_per_mole_nanometer
+};
+/**
+ * @brief Class converts between different unit types
+ */
+class UnitConverter {
+ private:
+  /// All distances with respect to Ang
+  constexpr double getDistanceValue_(const DistanceUnit& enum_type) const
+      noexcept {
+    switch (enum_type) {
+      case DistanceUnit::meters:
+        return 1E10;
+      case DistanceUnit::centimeters:
+        return 1E8;
+      case DistanceUnit::nanometers:
+        return 10.0;
+      case DistanceUnit::angstroms:
+        return 1.0;
+      case DistanceUnit::bohr:
+        return 0.529177;
```

Comment:
I am not sure if this is enough digits

Reply:
Ok will see if I can improve all the units, higher sf etc.

Reply:
This has been fixed.